### DISCUSSION TITLE
UK MOOSE Conference

### DISCUSSION THREAD

**Discussion by ABallisat** [idaholab/moose#29367] [Opportunities]
Hello all,

The Centre For Modelling & Simulation (CFMS) is planning to host a UK MOOSE conference in February 2025 at our offices in Bristol, UK. CFMS have been utilising MOOSE in a number of projects beyond the nuclear sector and want to bring together experts from academia, RTOs and industry to foster the UK MOOSE community. The idea is to have a combination of talks from across the user base, highlighting the industrial challenges faced and the capabilities of MOOSE, hopefully prompting discussions as to how MOOSE can be applied.

We are starting to put an agenda together for this event and gauging interest from the community. It would be really helpful to know if this is something you would be interested in attending or have any suggestions. Please feel free to reply to this or contact me directly.

Thanks in advance.

**Update January 2025**

The registration page is now live, date is the 27th February. Link to the registration page:

https://cfms.org.uk/article/2025-uk-moose-conference-register-your-interest/

**Comment by makeclean** [comment]
We'd love to come, we can send a few folks from UKAEA @helen-brooks @ajdubas @alexanderianblair

**Reply by alexanderianblair** [reply]
Yes, would be very keen!

**Reply by ajdubas** [reply]
Great idea, let me know as soon as you have a date as my February is booking up fast!

**Reply by helen-brooks** [reply]
ditto!

**Reply by ABallisat** [reply]
Great, I think I already have email addresses for you so will take this offline and you a message via email

**Comment by GiudGiud** [comment]
@ABallisat I emailed Rolls Royce (nuclear, they use some of our ec-ed tools) a few weeks ago to let them know but I did not hear back.
Happy to try again but I'd like to CC you so they have someone to talk to. Do you mind sharing your email at guillaume.giudicelli@inl.gov and I'll email when I get back

**Comment by ngrilli** [comment]
@ABallisat  thanks for organizing this.
There is a growing number of MOOSE users at the University of Bristol
and surely some of us can participate as we are close by.

**Comment by HyungseonSong-plasma** [comment]
Hi, I'm a scientist at Quantemol, a UK-based company specializing in plasma chemistry simulation software. Myself and a colleague are currently exploring MOOSE and would be very interested in attending this conference if it goes ahead. Please share details when they become available, thank you!

**Comment by ABallisat** [comment]
Thanks all for the interest. We are currently finalising the details and will have an event page soon which will cover sign ups as well. Once it is live I will post it on here.

**Comment by ABallisat** [comment]
Update: the registration page is now live, I have edited the original post with a link to the page, if anyone has any questions please just ask.
